native/src/components/CustomHeaderButtons.tsx

```diff
@@ -14,8 +14,8 @@ import styled from 'styled-components/native'
 import { LanguageIcon, MenuIcon, SearchIcon } from '../assets'
 import Icon from './base/Icon'
 
-const StyledOverflowMenu = styled(OverflowMenu)`
-  margin-right: 10px;
+const StyledHeaderContainer = styled.View`
+  margin: 0 10px 0 10px;
```

Comment:
Could be simplified to 
```suggestion
  margin: 0 10px;
```